examples/gallery/demos/bokeh/image_range_tool.ipynb

```diff
@@ -89,7 +89,7 @@
     "RangeToolLink(src, tgt, axes=['x', 'y'])\n",
     "\n",
     "(tgt + src).opts(\n",
-    "    opts.Image(default_tools=[]),\n",
+    "    opts.Image(default_tools=['wheel_zoom']),\n",
```

Comment:
Does it still default to the RangeTool after this change?

Reply:
Fixed in bf5ec9f